recipe/meta.yaml

```diff
@@ -72,3 +72,6 @@ extra:
     - Akshay-Venkatesh
     - jakirkham
     - TomAugspurger
+
+  variants:
+    docker_image: {{ docker_image }}
```

Comment:
Adds the variant to `extra` to ensure that conda-build will recognize this is used and create the full matrix.

Thanks @jjhelmus for the [suggestion]( https://gitter.im/conda-forge/conda-forge.github.io?at=5d3a0ed66f384439765c9114 ). 🙂